src/vivarium_public_health/metrics/mortality.py

```diff
@@ -7,121 +7,152 @@
 excess mortality in the simulation.
 
 """
+from collections import Counter
+from typing import Dict, List
+
+import pandas as pd
+
+from vivarium.framework.engine import Builder, ConfigTree
+from vivarium.framework.event import Event
+from vivarium.framework.population import PopulationView
+
 from vivarium_public_health.disease import DiseaseState, RiskAttributableDisease
-from vivarium_public_health.metrics.utilities import (
-    get_age_bins,
-    get_deaths,
-    get_person_time,
-    get_years_of_life_lost,
-)
+from vivarium_public_health.metrics.stratification import ResultsStratifier
 
 
 class MortalityObserver:
     """An observer for cause-specific deaths, ylls, and total person time.
 
-    By default, this counts cause-specific deaths, years of life lost, and
-    total person time over the full course of the simulation. It can be
-    configured to bin these measures into age groups, sexes, and years
-    by setting the ``by_age``, ``by_sex``, and ``by_year`` flags, respectively.
+    By default, this counts cause-specific deaths and years of life lost over
+    the full course of the simulation. It can be configured to add or remove
+    stratification groups to the default groups defined by a
+    :class:ResultsStratifier.
 
     In the model specification, your configuration for this component should
     be specified as, e.g.:
 
     .. code-block:: yaml
 
         configuration:
-            metrics:
+            observers:
                 mortality:
-                    by_age: True
-                    by_year: False
-                    by_sex: True
+                    exclude:
```

Comment:
instead of referring on line 29 to the default groups defined by ResultsStratifier, it might be nice to note what the defaults are here, so it's easy to know what might need to be `exclude`d?

or perhaps this is something that we do so often that that's unnecessary? i'm not sure

Reply:
oh --- hm, reading the `ResultsStratifier` i think you mean for the defaults to be customizable in a given project. If that's correct i retract the above comment.

Reply:
yes exactly. the defaults are configurable in the models themselves!